manifest.php

```diff
@@ -35,7 +35,7 @@
     'label' => 'Item core extension',
     'description' => 'TAO Items extension',
     'license' => 'GPL-2.0',
-    'version' => '6.4.0',
+    'version' => '6.4.1',
     'author' => 'Open Assessment Technologies, CRP Henri Tudor',
     'requires' => array(
         'taoBackOffice' => '>=3.0.0',
```

Comment:
It has to require at least generis 8.0.0 which introduced GenerisTestCase class

Reply:
fixed, thx :)